ios/MullvadVPN/View controllers/SelectLocation/SelectLocationViewController.swift

```diff
@@ -11,36 +11,26 @@ import MullvadTypes
 import RelayCache
 import UIKit
 
-final class SelectLocationViewController: UIViewController, UITableViewDelegate {
-    private var tableView: UITableView?
+final class SelectLocationViewController: UITableViewController {
```

Comment:
Turning the VC into a UITableViewController will make it less flexible if we want to add subviews to it. For example, the iPad version of the upcoming search bar should probably be added as such.